packages/client-farcaster/src/environment.ts

```
@@ -0,0 +1,140 @@
+        .default(ActionTimelineType.ForYou),
+});
+
+export type FarcasterConfig = z.infer<typeof farcasterEnvSchema>;
+
+function safeParseInt(
+    value: string | undefined | null,
+    defaultValue: number
+): number {
+    if (!value) return defaultValue;
+    const parsed = parseInt(value, 10);
+    return isNaN(parsed) ? defaultValue : Math.max(1, parsed);
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)